cmd/livepeer/livepeer.go

```diff
@@ -45,6 +48,14 @@ var (
 	ErrKeygen    = errors.New("ErrKeygen")
 	EthTxTimeout = 600 * time.Second
 
+	// The timeout for ETH RPC calls
+	ethRPCTimeout = 20 * time.Second
+
+	// The interval at which the block watcher polls for new blocks
+	blockWatcherPollingInterval = 1 * time.Second
```

Comment:
Wouldn't it perhaps be better to have a polling interval closer to the mainnet blocktime? We can always adjust this downward for testing locally. Wdyt?

Reply:
I agree that we should eventually change the polling interval to be closer to mainnet block time, but since we're still at a stage where we're running nodes in environments with fast block times (and potentially event auto-deploying builds to these environments as our devops setup improves) I think its ok to keep the polling interval as is for now and once we move to a stage where most of our testing is in environments with more realistic block times then we can switch. WDYT?

Reply:
Sounds good !